public/app/features/alerting/unified/components/rules/deleted-rules/DeletedRulesPage.tsx

```diff
@@ -0,0 +1,14 @@
+import { withPageErrorBoundary } from '../../../withPageErrorBoundary';
+import { AlertingPageWrapper } from '../../AlertingPageWrapper';
+
+import { DeletedRules } from './DeletedRules';
+
+function DeletedrulesPage() {
+  return (
+    <AlertingPageWrapper navId="alerts-trash" isLoading={false}>
```

Comment:
I think the UX could be better if we loaded the rules here and used `isLoading` property of `AlertingPageWrapper` to handle the loading state.
We wouldn't need to handle the loading state inside the `DeletedRules` component and it would give us better loading consistency